newIDE/app/src/AssetStore/InstallAsset.spec.js

```diff
@@ -350,54 +355,152 @@ describe('InstallAsset', () => {
     });
   });
 
-  describe('filterMissingExtensions', () => {
-    it('filters extensions that are not loaded ', () => {
+  const emptyExtensionShortHeader: ExtensionShortHeader = {
+    tier: 'reviewed',
+    shortDescription: '',
+    authorIds: [],
+    extensionNamespace: '',
+    fullName: '',
+    name: 'NotExistingExtension',
+    version: '1.0.0',
+    url: '',
+    headerUrl: '',
+    tags: [],
+    category: '',
+    previewIconUrl: '',
+    eventsBasedBehaviorsCount: 0,
+    eventsFunctionsCount: 0,
+  };
+
+  // TODO Find a way to test this
+
+  // describe('fetchAssets', () => {
+  //   it("throws if asset can't be downloaded", async () => {
+  //     mockFn(Asset.getPublicAsset).mockImplementationOnce(() => {
+  //       throw new Error('Fake error - unable to download');
+  //     });
+
+  //     const fetchAssets = useFetchAssets();
+  //     await expect(fetchAssets([fakeAssetShortHeader1])).rejects.toMatchObject({
+  //       message: 'Fake error - unable to download',
+  //     });
+
+  //     expect(getExtensionsRegistry).not.toHaveBeenCalled();
+  //     expect(getExtension).not.toHaveBeenCalled();
+  //   });
+  // });
```

Comment:
I'm not sure it's possible to test this, but it's probably not really important.

Reply:
Testing hooks is a bit hard/we don't have the setup for, but I think it's not a big deal indeed.